.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Tests and publishing
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  release:
+    types:
+    - created
+
+jobs:
+
+  tests:
+    name: Tests ${{ matrix.os }} - Node ${{ matrix.node }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        node:
+          - 12.x
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - name: Compile
+      run: npm install && npm run compile
+    - name: Run tests on Linux with xvfb-run
+      run: xvfb-run -a npm test
```

Comment:
Before, we ran `nmp run test` instead of `npm test`. Do they do the same? (Probably, unless one skips the pre-tests or something?)

Reply:
Good catch.  I copied this from [the documentation](https://code.visualstudio.com/api/working-with-extensions/continuous-integration#github-actions), but [I also found](https://github.com/giordano/nisaba-test/commit/fff5302776bd163e7369ca52db15181951588340) the documentation was wrong about the github ref to use for publishing......  According to [this answer of stackoverflow](https://stackoverflow.com/a/57031616), `npm test` and `npm run test` do the same thing, and indeed they both run the pretest script as well, so we can probably keep this?